bolt-contracts/src/contracts/BoltEigenLayerMiddleware.sol

```diff
@@ -29,7 +30,7 @@ contract BoltEigenLayerMiddleware is IBoltMiddleware, Ownable {
 
     /// @notice Validators registry, where validators are registered via their
     /// BLS pubkey and are assigned a sequence number.
-    IBoltValidators public validators;
+    BoltManager public boltManager;
```

Comment:
It would be great to use the interface instead of the contract here for consistency